packages/cashier/src/pages/account-transfer/account-transfer-no-account/account-transfer-no-account.tsx

```diff
@@ -3,9 +3,15 @@ import { Icon, Button, Text } from '@deriv/components';
 import { getPlatformSettings } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
+import { TRootStore, TClientStore, TUiStore } from 'Types';
 import './account-transfer-no-account.scss';
 
-const AccountTransferNoAccount = ({ toggleAccountsDialog, is_dxtrade_allowed }) => (
+type TAccountTransferNoAccount = {
```

Comment:
```suggestion
type TProps = {
```
It's a common convention to name the props type as `TProps` since they are local to this component only.
It's just a suggestion tho, You can ignore it 🙇🏻

Reply:
Same as above